packages/ui/src/containers/SocialLinkAccount/index.tsx

```diff
@@ -0,0 +1,57 @@
+import classNames from 'classnames';
+import { useTranslation } from 'react-i18next';
+
+import Button from '@/components/Button';
+import Divider from '@/components/Divider';
+import useBindSocialRelatedUser from '@/hooks/use-social-link-related-user';
+import useSocialRegister from '@/hooks/use-social-register';
+import type { SocialRelatedUserInfo } from '@/types/guard';
+import { maskEmail, maskPhone } from '@/utils/format';
+
+import * as styles from './index.module.scss';
+
+type Props = {
+  className?: string;
+  connectorId: string;
+  relatedUser: SocialRelatedUserInfo;
+};
+
+const SocialLinkAccount = ({ connectorId, className, relatedUser }: Props) => {
+  const { t } = useTranslation();
+
+  const bindSocialRelatedUser = useBindSocialRelatedUser();
+  const registerWithSocial = useSocialRegister(connectorId);
```

Comment:
nit: `bindWithSocial`? I saw you replaced `register` with `bind` earlier

Reply:
different cases, this hook returns the social register API precisely.